### ISSUE TITLE
"Implement members" intention doesn't trigger inside the impl block

### ISSUE THREAD

**Issue by Riateche** [intellij-rust/intellij-rust#3325]
When I create a trait impl, I have this situation:
```rust
impl Default for A {
     [cursor here]
}
```
The plugin suggests to implement members but I have to move cursor up to `impl Default for A` to use the suggestion. It would be much more convenient to trigger it from where the cursor is. Note that the suggestion doesn't appear until I type `impl Default for A {` and press Enter, so I *always* have to go back.

**Comment by Undin** [comment]
@Riateche But it's how platform annotations work - you can add some annotation (in this case error annotation) with an optional quick fix. And this quick fix can be invoked only when caret inside text range of the corresponding annotation. For example, similar error annotations for Java and Kotlin works in the same way.
Of course, it's possible to highlight full `impl` block to enable the quick fix in any position inside `impl` block but looks like it's rather annoying.

**Comment by vlad20012** [comment]
Btw I hate how it works for Java/Kotlin. I have to always move the cares to class definition to implement members

**Comment by vlad20012** [comment]
We can fix it if also register the quick fix as an intention

**Comment by Undin** [comment]
@Riateche @vlad20012 Looks like I've supported this feature in #3375
But `Implement methods` action should also help you as well

**Comment by Undin** [comment]
@Riateche Eventually we've decided that `Implement methods` action does exactly what you want and it doesn't make much sense to duplicate this functionality via quick fix/intention
So, I think it can be closed
